src/main/java/com/osiris/autoplug/client/tasks/updater/mods/TaskModsUpdater.java

```diff
@@ -249,15 +249,13 @@ else if (code == 3)
                     getWarnings().add(new BWarning(this, new Exception("Unknown error occurred! Code: " + code + "."), "Notify the developers. Fastest way is through discord (https://discord.gg/GGNmtCC)."));
 
                 try {
-                    YamlSection mmodrinthId = modsConfig.get(modsConfigName, mod.getName(), "modrinth-id");
-                    if (resultmodrinthId != null
-                            && (mmodrinthId.asString() == null || mmodrinthId.asInt() == 0)) // Because we can get a "null" string from the server
```

Comment:
If I leave the `== 0` checks, we get an exception here since AutoPlug writes a string value to the config such as `geyser-fabric` and the `asInt()` call throws since it's not a number. This results in a warning printed to the terminal. Before, it seems we assigned `mmodrinthId` to `result.getSpigotId()` which was never set so we never proceeded to call `mmodrinthId.asInt()`.